examples/connext_dds/asynchronous_publication_statistic/README.md

```diff
@@ -0,0 +1,44 @@
+# Example Code: Print QoS
+
+Example about getting asynchornous publisher statistic.
+
+## Concept
+
+RTI Connext DDS provides information about asynchronous publisher statistic data.
+
+| Structure | Field Name | Description |
+| --------- | ---------- | ----------- |
+| DDS_DataWriterProtocolStatus | pushed_sample_count | The number of user samples pushed on write from a local DataWriter to a matching remote DataReader. |
+| DDS_DataWriterProtocolStatus | pulled_sample_count | The number of user samples pulled from local DataWriter by matching DataReaders. |
+| DDS_DataWriterProtocolStatus | pushed_fragment_count | The number of DATA_FRAG messages that have been pushed by this DataWriter. |
+| DDS_DataWriterProtocolStatus | pulled_fragment_count | The number of DATA_FRAG messages that have been pulled from this DataWriter. |
+| DataReaderProtocolStatus | received_sample_count | The number of samples received by a DataReader. |
+| DataReaderProtocolStatus | received_fragment_count | The number of DATA_FRAG messages that have been received by this DataReader. |
+
+For more information about the counters, please refer to the API Reference HTML
+documentation or the User Manuals.
+
+## Example Description
+
+In this example, we illustrate how to access the asynchronous publisher statistic:
+
+In order to update the instance statistic we have write several sample using
```

Comment:
typo instance statistic -> fragmented data statistics (or asynchronous publisher statistics, pending decision from Jose  / Fernando)

In order to update the <> statistic**s** we have **to** write sever sample**s** that are fragmented. <Maybe include here a sentence about when data fragmentation occurs)